harvard-university-of-leeds.csl

```diff
@@ -32,7 +36,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="text" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
+      <name and="text" delimiter-precedes-last="never" et-al-min="0" initialize-with="." name-as-sort-order="all"/>
```

Comment:
Yes, that, and it also means you don't inherit a different setting set at a higher level.